#!/usr/bin/env python
PACKAGE = "hektar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("claw_theta", int_t, 0, "claw target theta value", 0, -90, 90)
gen.add("claw_r", int_t, 0, "claw target r value", 300, 10, 500)
gen.add("claw_z", int_t, 0, "claw target z value", 150, 0, 350)

exit(gen.generate(PACKAGE, "hektar", "ClawSlider"))
